pymoskito/simulation_modules.py

```diff
@@ -67,6 +70,24 @@ def __init__(self, settings):
         self._settings["step width"] = None
         self._settings.pop("modules", None)
 
+        if settings.get("Module", None) is not None:
```

Comment:
I think, if the YIT compilation is needed, the `generate_binding` should be split into `create_binding_config` and `build_binding` . Then a mix-in class like `CppBinding` can be added which adds this functionality and is provided with the required information. Afterwards you can  wirte something along the lines of `class PIDController(pm.Controller,  pm.CppBinding):` in your code.
If you do not dislike this approach to much, I'll provide an implementation of that idea.

Reply:
Yes I think the YIT sould be avaiable and I like your idea with the CppBinding Class. So it can moved the 'generate_binding i